Sources/Paywalls/Components/PaywallButtonComponent.swift

```diff
@@ -21,13 +21,38 @@ public extension PaywallComponent {
 
     struct ButtonComponent: PaywallComponentBase {
 
+        // swiftlint:disable:next nesting
+        public enum Action: Codable, Sendable, Hashable, Equatable {
+            case restorePurchases
+            case navigateTo(destination: Destination)
+            case navigateBack
+        }
+
+        // swiftlint:disable:next nesting
+        public enum Destination: Codable, Sendable, Hashable, Equatable {
+            case customerCenter
+            case URL(urlLid: String, method: URLMethod)
+            case privacyPolicy(urlLid: String, method: URLMethod)
+            case terms(urlLid: String, method: URLMethod)
```

Comment:
These all have a `urLLid` property to follow our convention of passing "localization IDs". Let me know if this is not the way we should localize URLs.

Reply:
This makes sense to me! 👍